crates/swc_ecma_minifier/src/compress/optimize/util.rs

```diff
@@ -102,6 +102,39 @@ impl<'b> Optimizer<'b> {
             orig_ctx,
         }
     }
+
+    pub(super) fn try_remove_label(&mut self, s: &mut LabeledStmt) {
+        if !self.options.dead_code {
+            return;
+        }
+
+        let mut analyer = LabelAnalyzer {
+            label: s.label.sym.clone(),
+            ..Default::default()
+        };
+
+        match &mut *s.body {
+            Stmt::For(ForStmt { body, .. })
+            | Stmt::ForIn(ForInStmt { body, .. })
+            | Stmt::ForOf(ForOfStmt { body, .. })
+            | Stmt::While(WhileStmt { body, .. })
+            | Stmt::DoWhile(DoWhileStmt { body, .. }) => {
+                analyer.top_breakable = true;
+                body.visit_mut_with(&mut analyer)
+            }
+            Stmt::Switch(SwitchStmt { cases, .. }) => {
+                analyer.top_breakable = true;
+                cases.visit_mut_with(&mut analyer)
+            }
+            _ => s.body.visit_mut_with(&mut analyer),
```

Comment:
Actually thinking about it some more, do we even need to run this if we're not a loop/switch? We can't remove `break` statements, and we know a `continue` statement can't exist. Can we just early return here and remove `top_breakable` entirely?

Reply:
We still need to find out if this label is ever used.